code/datums/traits/good.dm

```diff
@@ -96,6 +96,33 @@
 	)
 	H.equip_in_one_of_slots(B, slots , qdel_on_fail = TRUE)
 
+/datum/quirk/multilingual
+	name = "Multilingual"
+	desc = "You spent a portion of your life learning to understand an additional language. You may or may not be able to speak it based on your anatomy."
+	value = 2
+	mob_trait = TRAIT_MULTILINGUAL
+	gain_text = "<span class='notice'>You have learned to understand an additional language.</span>"
+	lose_text = "<span class='danger'>You have forgotten how to understand a language.</span>"
+
+/datum/quirk/multilingual/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/datum/language_holder/LH
```

Comment:
Bacon was right, this appears to have never been initialized

Reply:
Yeah I realized that fuck up. I’ll get that changed this evening when I get off my shift. Expect a commit for that eventually.

Reply:
So I did some more refactoring and scrapped LH entirely.